### PR TITLE
Disable logged in users from accessing the login page

### PR DESCRIPTION
**Pull request by FelixWambiri**
**Description of Task to be completed?**
* Disable logged in Users from accessing the login page

**How should this be manually tested?**
* Login with any account
* Click on the back button

**What are the relevant pivotal tracker stories?**
* #157840079

**Background context**
* At the moment when a logged in user clicks on the back button they can access the login page.

### CODE REVIEW

**Review by mutoromike — APPROVED** [review]

**Review by meshnesh — APPROVED** [review]
Nice implementation and completing the task

**Review by betsybeth — APPROVED** [review]
Good work.LGTM

**Review by Kalela — APPROVED** [review]
LGTM

File: wger/core/views/user.py [hunk 1]
```diff
@@ -73,6 +73,12 @@ def login(request):
     '''
     Small wrapper around the django login view
     '''
+    try:
```
Comment (frankip) [inline wger/core/views/user.py:76]: good job 
work fine

**Review by frankip — CHANGES_REQUESTED** [review]
Dont push the settings file
add the settings to gitignore

File: wger/core/views/user.py (hunk 1 above)
Comment (FelixWambiri) [reply wger/core/views/user.py:76]: Thanks,I have deleted the settings file

**Pull request merged**